### ISSUE TITLE
DN size to inner and outer diameter conversion

### ISSUE THREAD

**Issue by vanmeerkerk** [Nieuwe-Warmte-Nu/simulator-core#43]
How are we going to implement the conversion step for items that only have a DN-size?

**Comment by vanmeerkerk** [comment]
@octavianor maybe only support innderDiameter and put the responsibility somewhere else?

**Comment by vanmeerkerk** [comment]
@samvanderzwan hoe werkt dit in ESDL? Anders kan Ashraf dit implementeren.

**Comment by vanmeerkerk** [comment]
@samvanderzwan @octavianor  is this something @AshrafAlmohagryDeltares  can pick up?

**Comment by samvanderzwan** [comment]
Yes I think so, check with optimizer team how they implemented this

**Comment by vanmeerkerk** [comment]
@AshrafAlmohagryDeltares will catch-up with Kobus or Femke to check how it is implemented.

**Comment by AshrafAlmohagryDeltares** [comment]
I checked with Kobus. They defined the inner diameters based on the DN of the pipe from list at this website (https://edr.hesi.energy/cat/Assets). 

In other words, in the current implementation, the inner diameter is retrieved from a predefined dictionary based on the pipe's DN. This means the user only needs to select the pipe DN, and the corresponding inner diameter is automatically selected from the list within the code.
